delivery_carrier_seur/model/stock.py

```diff
@@ -0,0 +1,167 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 FactorLibre (http://www.factorlibre.com)
+#                  Ismael Calvo <ismael.calvo@factorlibre.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import logging
+from unidecode import unidecode
+from urllib2 import HTTPError
+
+from openerp import models, fields, api, exceptions
+from openerp.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+try:
+    from seur.picking import Picking
+except ImportError:
+    _logger.debug('Can not `from seur.picking import Picking`.')
+
+
+class ShippingLabel(models.Model):
+    _inherit = 'shipping.label'
+
+    @api.model
+    def _get_file_type_selection(self):
+        """ To inherit to add file type """
+        res = super(ShippingLabel, self)._get_file_type_selection()
+        res.append(('txt', 'TXT'))
+        return res
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    seur_service_code = fields.Selection(
```

Comment:
You may use carrier option mechanism to have a cleaner interface
https://github.com/OCA/carrier-delivery/blob/8.0/delivery_carrier_label_postlogistics/delivery.py#L76